tests/Microsoft.Graph.DotnetCore.Core.Test/Requests/BaseRequestBuilderTests.cs

```diff
@@ -32,5 +37,107 @@ public void AppendSegmentToRequestUrl()
 
             Assert.Equal(string.Join("/", requestUrl, newUrlSegment), appendedUrl);
         }
+
+        /// <summary>
+        /// Test that: 1) composable functions have all parameters set, and 2) query options are not yet applied to the URL.
+        /// </summary>
+        [Fact]
+        public void ComposableFunctionTest()
+        {
+            var requestUrl = "https://localhost:443/microsoft.graph.composablefunction0";
```

Comment:
requestURL, client can be stated as constants and reused in multiple tests.

Reply:
Fixed